spec/message.abnf

```diff
@@ -55,7 +55,11 @@ name-start = ALPHA / "_"
            / %x2070-218F / %x2C00-2FEF / %x3001-D7FF
            / %xF900-FDCF / %xFDF0-FFFD / %x10000-EFFFF
 name-char = name-start / DIGIT / "-" / "." / ":"
-          / %xB7 / %x0300-036F / %x203F-2040
+          / %xB7 / %x300-36F / %x203F-2040
+
+unquoted = unquoted-start *name-char
```

Comment:
I'd like to bikeshed the name a bit. `unquoted` suggested that it's an *unquoted literal*, but in fact, it's much more limited than that. It sits somewhere between `name` and `nmtoken`.

Reply:
Suggestions are welcome. I started with `argtoken`, but renamed to `unquoted` on @aphillips's request.